tests/utils.go

```diff
@@ -2076,3 +2084,43 @@ func GetDefaultVirtHandlerDaemonSet(namespace string, config *util.KubeVirtDeplo
 func GetDefaultExportProxyDeployment(namespace string, config *util.KubeVirtDeploymentConfig) (*v12.Deployment, error) {
 	return components.NewExportProxyDeployment(namespace, config.GetImageRegistry(), config.GetImagePrefix(), config.GetExportProxyVersion(), "", "", "", config.VirtExportProxyImage, config.GetImagePullPolicy(), config.GetImagePullSecrets(), config.GetVerbosity(), config.GetExtraEnv())
 }
+
+func GetDaemonSet(virtCli kubecli.KubevirtClient, componentName, namespace string) *v12.DaemonSet {
+	daemonSet, err := virtCli.AppsV1().DaemonSets(namespace).Get(context.Background(), componentName, metav1.GetOptions{})
+	ExpectWithOffset(1, err).NotTo(HaveOccurred())
+
+	return daemonSet
+}
+
+func GetReadyDaemonSetReplicas(virtCli kubecli.KubevirtClient, componentName, namespace string) func() int32 {
+	return func() int32 {
+		replicas := GetDaemonSet(virtCli, componentName, namespace).Status.NumberReady
+		return replicas
+	}
+}
+
+func GetDesiredDaemonSetReplicas(virtCli kubecli.KubevirtClient, componentName, namespace string) int32 {
+	return GetDaemonSet(virtCli, componentName, namespace).Status.DesiredNumberScheduled
+}
+
+func BlackHolePod(getter GetterPodsFunc, virtCli kubecli.KubevirtClient, componentName, namespace, nodeName string, addOrDel BlackHoleAction) {
```

Comment:
Can we have funcs with proper names to decide what to do instead of passing `addOrDel` strings? Can we move this func to `libpod`?

Reply:
Done.